src/Pmi/Drc/RdrHelper.php

```diff
@@ -54,10 +54,12 @@ public function getClient($resourceEndpoint = null)
             $endpoint = $this->endpoint;
         }
 
-        $client = new \Memcache();
-        $cachePool = new MemcacheCachePool($client);
+        if (class_exists('Memcache')) {
```

Comment:
Did you mean to include this change? I suppose it makes sense, but I didn't see anything in the commit message that indicates this.

Reply:
No, it wasn't intentional. I needed it for debugging, and wasn't specific enough in my `git add src/Pmi` command.